testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -212,3 +236,80 @@ fn block_proposal_rejection() {
     }
     signer_test.shutdown();
 }
+
+// Basic test to ensure that miners are able to gather block responses
+// from signers and create blocks.
+#[test]
+#[ignore]
+fn miner_gather_signatures() {
+    if env::var("BITCOIND_TEST") != Ok("1".into()) {
+        return;
+    }
+
+    tracing_subscriber::registry()
+        .with(fmt::layer())
+        .with(EnvFilter::from_default_env())
+        .init();
+
+    info!("------------------------- Test Setup -------------------------");
+    let num_signers = 5;
+    let mut signer_test: SignerTest<SpawnedSigner> = SignerTest::new(num_signers);
+    signer_test.boot_to_epoch_3();
+    let timeout = Duration::from_secs(30);
+
+    info!("------------------------- Try mining one block -------------------------");
+    signer_test.mine_nakamoto_block(timeout);
+
+    // Verify that the signers accepted the proposed block, sending back a validate ok response
+    let proposed_signer_signature_hash = signer_test.wait_for_validate_ok_response(timeout);
+    let message = proposed_signer_signature_hash.0;
+
+    info!("------------------------- Test Block Signed -------------------------");
+    // Verify that the signers signed the proposed block
+    let signature =
+        signer_test.wait_for_confirmed_block_v0(&proposed_signer_signature_hash, timeout);
+
+    info!("Got {} signatures", signature.len());
+
+    assert_eq!(signature.len(), num_signers);
+
+    let reward_cycle = signer_test.get_current_reward_cycle();
+    let signers = signer_test.get_reward_set_signers(reward_cycle);
+
+    // Verify that the signers signed the proposed block
+
+    let all_signed = signers.iter().zip(signature).all(|(signer, signature)| {
+        let stacks_public_key = Secp256k1PublicKey::from_slice(signer.signing_key.as_slice())
+            .expect("Failed to convert signing key to StacksPublicKey");
+
+        // let valid = stacks_public_key.verify(message, signature);
+        let valid = stacks_public_key
+            .verify(&message, &signature)
+            .expect("Failed to verify signature");
+        if !valid {
+            error!(
+                "Failed to verify signature for signer: {:?}",
+                stacks_public_key
+            );
+        }
+        valid
+    });
+    assert!(all_signed);
+
+    // Test prometheus metrics response
+    #[cfg(feature = "monitoring_prom")]
```

Comment:
It this getting tested in CI?

Reply:
This test is running in the `integration-tests` workflow, and I believe those tests do include the `monitoring_prom` flag enabled.